ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/log/FootstepPlannerLogger.java

```diff
@@ -49,21 +50,23 @@ public class FootstepPlannerLogger
    private static final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyyMMdd_HHmmssSSS");
    private static final int RECOMMENDED_NUMBER_OF_LOGS_TO_KEEP = 500;
    public static final String defaultLogsDirectory;
+
    static
    {
-      String incomingLogsDirectory = System.getProperty("user.home") + File.separator + ".ihmc" + File.separator;
+      String incomingLogsDirectory;
       if (ContinuousIntegrationTools.isRunningOnContinuousIntegrationServer())
       {
-         incomingLogsDirectory = incomingLogsDirectory + "bamboo-logs" + File.separator
-                                                       + System.getenv("bamboo_planKey") + File.separator
-                                                       + System.getenv("bamboo_buildResultKey") + File.separator;
+         incomingLogsDirectory =
+               System.getProperty("user.home") + File.separator + ".ihmc" + File.separator + "bamboo-logs" + File.separator + System.getenv("bamboo_planKey")
+               + File.separator + System.getenv("bamboo_buildResultKey") + File.separator;
       }
       else
       {
-         incomingLogsDirectory = incomingLogsDirectory + "logs" + File.separator;
+         incomingLogsDirectory = IHMCCommonPaths.ASTAR_FOOTSTEP_PLANNER_DIRECTORY.toString();
```

Comment:
If we aren't logging to bamboo (should fix that too lol) then use the default path.

Reply:
Yeah the bamboo logging can be removed in a later pr